tests/fingerprints/physiochemical_properties.py

```diff
@@ -0,0 +1,100 @@
+import numpy as np
+from scipy.sparse import csr_array
+
+from skfp.fingerprints import PhysiochemicalPropertiesFingerprint
+
+
+def test_physiochemical_properties_bp_bit_fingerprint(smiles_list):
```

Comment:
Didn't we settle for removing the "bit" part of test names?

Reply:
Well, here we have both bit and count variants, so I left this, to be explicit. Only those fingerprints that have only a single variant don't have this in test names.